assets/add_new_asset.md

```diff
@@ -139,20 +185,26 @@ https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/bitcoin/
 
 ERC20:
 ```js
-https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x006bea43baa3f7a6f765f14f10a1a1b08334ef45/logo.png
+https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x006BeA43Baa3f7A6f765F14f10A1a1b08334EF45/logo.png
 ```
 
 BEP-2:
 ```js
-https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/assets/ankr-e97/logo.png
+https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/binance/assets/ANKR-E97/logo.png
 ```
 
 TRC-10:
 ```js
 https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tron/assets/1002000/logo.png
 ```
 
+TRC-20:
+```js
+https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tron/assets/TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t/logo.png
+```
+
 ## Used in Applications
 - [Trust Wallet](https://trustwallet.com) - [iOS](https://itunes.apple.com/us/app/trust-ethereum-wallet/id1288339409) and [Android](https://play.google.com/store/apps/details?id=com.wallet.crypto.trustapp)
-- [0x Tracker](https://0xtracker.com) - The 0x Protocol Trade Explorer and news aggregator.
-
```

Comment:
no need to include this here, only keep `## Used in Applications` on github

Reply:
https://github.com/trustwallet/developer/pull/63/commits/53df752447fd2e3a8b755a2dff75b3b90fea70d4